develocity-gradle.yml

```diff
@@ -200,65 +224,133 @@ spec:
+                                  { develocity },
+                                  { buildScan }
+                          )
+                          def buildScanExtension = dvOrGe(
+                                  { rootExtension.buildScan },
+                                  { rootExtension }
+                          )
+                          if (develocityUrl) {
+                              logger.lifecycle("Connection to Develocity: $develocityUrl, allowUntrustedServer: $develocityAllowUntrustedServer, captureFileFingerprints: $develocityCaptureFileFingerprints")
+                              rootExtension.server = develocityUrl
+                              rootExtension.allowUntrustedServer = develocityAllowUntrustedServer
+                          }
+                          if (!shouldApplyDevelocityPlugin) {
+                              // Develocity plugin publishes scans by default
+                              buildScanExtension.publishAlways()
+                          }
+                          // uploadInBackground not available for build-scan-plugin 1.16
+                          if (buildScanExtension.metaClass.respondsTo(buildScanExtension, 'setUploadInBackground', Boolean)) buildScanExtension.uploadInBackground = buildScanUploadInBackground
+                          buildScanExtension.value CI_AUTO_INJECTION_CUSTOM_VALUE_NAME, ciAutoInjectionCustomValueValue
+                          if (isAtLeast(develocityPluginVersion, '2.1') && atLeastGradle5) {
+                              if (isAtLeast(develocityPluginVersion, '3.17')) {
+                                  buildScanExtension.capture.fileFingerprints = develocityCaptureFileFingerprints
+                              } else if (isAtLeast(develocityPluginVersion, '3.7')) {
+                                  buildScanExtension.capture.taskInputFiles = develocityCaptureFileFingerprints
                               } else {
-                                  buildScan.captureTaskInputFiles = geCaptureTaskInputFiles
+                                  buildScanExtension.captureTaskInputFiles = develocityCaptureFileFingerprints
                               }
                           }
                       }
 
-                      if (geUrl && geEnforceUrl) {
-                          pluginManager.withPlugin(BUILD_SCAN_PLUGIN_ID) {
-                              afterEvaluate {
-                                  logger.lifecycle("Enforcing Gradle Enterprise: $geUrl, allowUntrustedServer: $geAllowUntrustedServer")
-                                  buildScan.server = geUrl
-                                  buildScan.allowUntrustedServer = geAllowUntrustedServer
+                      if (develocityUrl && develocityEnforceUrl) {
+                          logger.lifecycle("Enforcing Develocity: $develocityUrl, allowUntrustedServer: $develocityAllowUntrustedServer, captureFileFingerprints: $develocityCaptureFileFingerprints")
```

Comment:
We don't need to output `develocityCaptureFileFingerprints` as it should be set only if the Dv plugin is not already applied in the project

Reply:
Thank you for spotting this! Fixed in https://github.com/gradle/develocity-gitlab-templates/pull/25/commits/41f7a89ea28c0773d451d46ccce02701d6bf074b